service-templates/load-balanced-fargate-svc/README.md

```diff
@@ -31,6 +31,12 @@ The template also provisions a CodePipeline based pipeline to pull your applicat
 3. unit_test_command: The command to run to unit test the application code
 4. environment_account_ids: The environment account ids for service instances using cross account environment
 
+### (Optional) Extend the Service with a Component
+
+The service template which you used above will provision a Fargate service with the infrastructure required for networking, monitoring and scaling the service. In practice you may want to use a template like this for many different purposes and can use components to extend the service template to meet your requirements.
+
+Jinja filters will be used to optionally incorporate component outputs of our s3-bucket-component sample which provisions an S3 bucket and defines an IAM Policy for accessing that bucket will be associated with the service instance.
+
```

Comment:
This paragraph doesn't read cleanly, to me. For the Terraform template I added this:

```
The s3-bucket-component sample provisions an S3 bucket and defines an IAM Policy for accessing it. If present, the bucket policy and bucket ARN outputs from the s3-bucket-component will be used in the ECS task role and ECS environment variables, respectively.
```

Reply:
Updated